docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Introducing MUI Base: the headless alternative to Material UI'
+description: 'Introduction to a new library of unstyled components and hooks: MUI Base'
+date: 2022-08-29T00:00:00.000Z
+authors: ['michaldudak', 'samuelsycamore']
+tags: ['News', 'MUI Core']
+card: true
+---
+
+<img src="/static/blog/introducing-mui-base/hero-image.png" style="width: 796px; margin-bottom: 24px;" alt="Demo components built with MUI Base, a newly introduced library of unstyled components and hooks" />
```

Comment:
Set the correct width of the image (though we could also remove it as it's always full width) and set the aspect ratio to avoid layout jumps.

```suggestion
<img src="/static/blog/introducing-mui-base/hero-image.png" style="width: 692px; aspect-ratio: 132/61; margin-bottom: 24px;" alt="Demo components built with MUI Base, a newly introduced library of unstyled components and hooks" />
```